pkgs/by-name/ml/mlkit/package.nix

```
@@ -17,6 +17,9 @@ stdenv.mkDerivation rec {
 
   doCheck = true;
 
+  # MLKit intentionally has some of these in its test suite.
```

To clarify:  the `no-broken-symlinks` hook runs on the outputs of the derivation. The reason the hook fails the build isn't because the test suite contains broken symlinks (https://github.com/melsman/mlkit/blob/1733d3d90fc3ebd6157e1c34bcd68de51ab0d722/test/README#L13-L14), but rather because the test suite is copied to `$out/share/mlkit/test` (https://github.com/melsman/mlkit/blob/1733d3d90fc3ebd6157e1c34bcd68de51ab0d722/Makefile.in#L228).

If there's a reason we want to keep the test suite around in the output, I suggest the following:

```suggestion
  # MLKit intentionally has some of these in its test suite.
  # Since the test suite is available in `$out/share/mlkit/test`, we must disable this check.
```